cmd/root.go

```diff
@@ -31,7 +31,9 @@ var (
 
 		$ stellar auth activate-api-key path/to/stellarstation-private-key.json
 
-		All commands should work after that.`)
+		All commands should work after that.
+
+		Notice: The open stream flag '--enable-auto-close' which is supplied with an '--auto-close-time' has been deprecated and will be removed in a future release.'`)
```

Comment:
I think the notice only upon usage and within the command documentation (generated) is fine. We can talk about the deprecation in more detail when we release the next version containing this change 🌆

Reply:
If I remove the option from the --help text, and generate the documentation again, it will be removed here as well.

Reply:
In that case you can put `deprecated` on the related auto-close flags similar to what we did for the old proxy flags

https://github.com/infostellarinc/stellarcli/blob/691395b530498274f501480d9dde194508eda47b/cmd/flag/proxy_flags.go#L66-L69